src/Kotlin/DocumentationBuilder.kt

```diff
@@ -128,7 +175,11 @@ class DocumentationBuilder(val session: ResolveSession, val options: Documentati
         for ((packageName, declarations) in descriptors) {
             println("  package $packageName: ${declarations.count()} nodes")
```

Comment:
Changed "nodes" to "declarations" so that the number remains correct. We can refactor to start using a proper logging framework later. (And no slf4j please - I'd rather use JDK logging.)